docs/recipes/WritingTests.md

```diff
@@ -79,7 +79,7 @@ export default function todos(state = initialState, action) {
   switch (action.type) {
   case ADD_TODO:
     return [{
-      id: state.length,
+      id: state.reduce((maxId, todo) => Math.max(todo.id, maxId), -1) + 1,
```

Comment:
When maxId is not declared then it should have the value as **undefined**
however its getting the value as **null** ?